# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../blackdown-jre/extract-x86.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- j2re-1.4.2-01-linux-i586.bin	2005-03-24 12:03:57.000000000 +0100
+++ j2re-1.4.2-01-linux-i586.bin	2005-03-24 12:26:21.000000000 +0100
@@ -3,7 +3,7 @@
 # This script was generated using Makeself 1.5.5-Blackdown-j2se with modifications from Blackdown (source code is available from Blackdown mirrors)
 TAR_CMD=tar
 CRCsum=1315459154
-MD5=e2dd4b31085b43d2deae62f22ba14bcc
+MD5=\"00000000000000000000000000000000\"; SETUP_NOCHECK=1
 PATH=/usr/bin:/bin
 label="Blackdown Java 2 Standard Edition Runtime Environment v1.4.2-01 for Linux/i586"
 NO_DOT_VERSION=142
@@ -112,7 +112,7 @@
         fi
     fi
 fi
-more << EOF_LICENSE
+agreed=1; cat << EOF_LICENSE
 Sun Microsystems, Inc. Binary Code License Agreement
 for the JAVATM 2 RUNTIME ENVIRONMENT (J2RE), STANDARD
 EDITION, VERSION 1.4.2_X
